processor/cumulativetodeltaprocessor/go.mod

```diff
@@ -0,0 +1,9 @@
+module github.com/open-telemetry/opentelemetry-collector-contrib/processor/cumulativetodeltaprocessor
+
+go 1.16
+
+require (
+	github.com/stretchr/testify v1.7.0
+	go.opentelemetry.io/collector v0.29.1-0.20210629183146-cbee3b39483c
```

Comment:
Update this to the same version as in others, need to have a replace statement for pdata :)